### PR TITLE
addmaintainer, increase build number

### PR DESCRIPTION
**Pull request by Tata17**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by marcelotrevisani** [comment]
I was just checking this recipe with [grayskull](https://github.com/marcelotrevisani/grayskull) and just updated with some small improvements
I hope you don't mind

**Comment by Tata17** [comment]
> grayskull

of course not, for the contrary, thank you!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,16 +31,23 @@ requirements:
 test:
   imports:
     - hyperas
+  commands:
+    - pip check
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: ow, that is interesting, ``pip check`` is not working well for this package...
at least it is not detecting ``jupyter``

**Pull request merged**